src/Components/IssuesControl.jsx

```diff
@@ -191,23 +189,25 @@ export function Issues() {
       try {
         const commentsArr = []
         const commentsData = await getComments(repository, selectedIssue.number)
-        commentsData.map((comment) => {
-          const lines = comment.body.split('\r\n')
-          const embeddedUrl = lines.filter((line) => line.includes('url'))[0]
-          const commentImageUrl = comment.body.split('ImageUrl')[1]
-          const body = lines[0]
-          commentsArr.push({
-            embeddedUrl: embeddedUrl,
-            id: comment.id,
-            number: comment.number,
-            title: comment.title,
-            body: body,
-            date: comment.created_at,
-            username: comment.user.login,
-            avatarUrl: comment.user.avatar_url,
-            imageUrl: commentImageUrl,
+        if (commentsData) {
```

Comment:
@pablo-mayrgundter - this PR is about this if statement. 
commentsData is undefined when no comments were attached to the issue, which was causing the error int he pull logic and the state would never reset.